<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpPage</class>
 <widget class="QWidget" name="SignUpPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>30</number>
    </property>
    <property name="topMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>30</number>
    </property>
    <property name="bottomMargin">
     <number>30</number>
    </property>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Role</string>
      </property>
      <property name="buddy">
       <cstring>RoleCombo</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="UserIdLine"/>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="SignUpButton">
      <property name="text">
       <string>Sign up!</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="UserNameLine"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>User Name</string>
      </property>
      <property name="buddy">
       <cstring>UserNameLine</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="PassWordLine"/>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="GoBackButton">
      <property name="text">
       <string>Go Back</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Password</string>
      </property>
      <property name="buddy">
       <cstring>PassWordLine</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>User ID</string>
      </property>
      <property name="buddy">
       <cstring>UserIdLine</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="RoleCombo">
      <item>
       <property name="text">
        <string>Student</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Teacher</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
